js-rattler/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "@baszalmstra/rattler",
+    "name": "@conda-org/rattler",
```

Comment:
Unfortunately we dont have consensus within the steering council yet where this package should go, so for now we should keep it under my username.

Reply:
ok